google/cloud/security/inventory/pipelines/load_org_iam_policies_pipeline.py

```diff
@@ -49,53 +48,101 @@ def __init__(self, cycle_timestamp, configs, crm_rate_limiter, dao):
+                    yield {'org_id': iam_policy_map['org_id'],
+                           'role': role,
+                           'member_type': member_type,
+                           'member_name': member_name,
+                           'member_domain': member_domain}
 
+    def _retrieve(self, org_id):
+        """Retrieve the org IAM policies from GCP.
+
+        Args:
+            org_id: String of the organization id
+
+        Returns:
+            iam_policies_map: List of IAM policies as per-org dictionary.
+                Example: {org_id: org_id,
+                          iam_policy: iam_policy}
+                https://cloud.google.com/resource-manager/reference/rest/Shared.Types/Policy
+        """
         try:
             # Retrieve data from GCP.
             # Flatten the iterator since we will use it twice, and it is faster
             # than cloning to 2 iterators.
-            iam_policies_map = self.gcp_api_client.get_org_iam_policies(
-                self.name, org_id)
-            # TODO: Investigate improving so the pylint disable isn't needed.
-            # pylint: disable=redefined-variable-type
-            iam_policies_map = list(iam_policies_map)
-
-            # Flatten and relationalize data for upload to cloud sql.
-            flattened_iam_policies = (
-                self.transform_util.flatten_iam_policies(iam_policies_map))
+            return list(self.gcp_api_client.get_org_iam_policies(
+                self.name, org_id))
         except ApiExecutionError as e:
-            raise LoadDataPipelineError(e)
-    
-        # Load flattened iam policies into cloud sql.
-        # Load raw iam policies into cloud sql.
-        # A separate table is used to store the raw iam policies because it is
-        # much faster than updating these individually into the projects table.
-        try:
-            self.dao.load_data(self.name, self.cycle_timestamp,
-                               flattened_iam_policies)
-    
-            for i in iam_policies_map:
-                i['iam_policy'] = json.dumps(i['iam_policy'])
-            self.dao.load_data(self.raw_org_iam_policies, self.cycle_timestamp,
-                               iam_policies_map)
-        except (CSVFileError, MySQLError) as e:
-            raise LoadDataPipelineError(e)
+            raise inventory_errors.LoadDataPipelineError(e)
+
+    def run(self):
+        """Runs the data pipeline."""
+        org_id = self.configs.get('organization_id')
+        # Check if the placeholder is replaced in the config/flag.
+        if org_id == '<organization id>':
+            raise inventory_errors.LoadDataPipelineError(
+                'No organization id is specified.')
+
+        iam_policies_map = self._retrieve(org_id)
+
+        flattened_iam_policies = self._flatten(iam_policies_map)
+
+        self._load(iam_policies_map, flattened_iam_policies)
+
+        super(LoadOrgIamPoliciesPipeline, self)._get_loaded_count()
```

Comment:
You can directly call self._get_loaded_count(), you shouldn't need to call up to super, that's only needed if you override _get_loaded_count() in the child class.